packages/shared/components/inputs/Dropdown.svelte

```diff
@@ -217,7 +230,7 @@
     bg-white dark:bg-gray-800 focus:border-blue-500 {dropdown ? 'border-blue-500' : showBorderWhenClosed ? 'border-gray-300 dark:border-gray-700 hover:border-gray-500 dark:hover:border-gray-700' : ''}"
         tabindex="0"
         bind:this={divContainer}>
-        <div class="w-full text-12 leading-140 text-gray-800 dark:text-white"><Text type={valueTextType} smaller>{value || placeholder}</Text></div>
+        <div class="w-full text-12 leading-140 text-gray-800 dark:text-white"><Text type={valueTextType} smaller>{search || value || placeholder}</Text></div>
```

Comment:
With the possibility to enter values now this issue has popped up 🕵🏼 
![image](https://user-images.githubusercontent.com/3624944/148973862-9543ba16-0cbf-4cce-b009-e67c0f1ab93f.png)